Makefile

```diff
@@ -152,3 +152,27 @@ runserver: ## Run the Django application locally
 
 .PHONY: dev-init
 dev-init: load-design-system-templates collectstatic makemigrations migrate createsuperuser ## Run the pre-run setup scripts
+
+.PHONY: functional-tests-up
+functional-tests-up:  ## Start the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose.yml up -d
+
+.PHONY: functional-tests-up-dev
+functional-tests-up-dev:  ## Start the functional tests docker compose dependencies and dev app
+	docker compose -f functional_tests/docker-compose-dev-app.yml up -d
+
+.PHONY: functional-tests-down
+functional-tests-down:  ## Stop the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose-dev-app.yml down
+
+.PHONY: functional-tests-run
+functional-tests-run: collectstatic  ## Run the functional tests
```

Comment:
This also requires the backing services to be up, so chain the call?. DS templates has to be loaded as well, it would be good for commands to not assume they would be running against a containerised app.

Reply:
The `functional-tests` target combines the setup and test run, the purpose of this command is to allow the dev to easily run the tests without the backing containers being torn down post-test.

Reply:
Right I see, then in that case, we just need to ensure DS templates are loaded.